projects/github-actions/pr-is-up-to-date/composer.json

```diff
@@ -7,6 +7,9 @@
 	"require-dev": {
 		"automattic/jetpack-changelogger": "3.0.x-dev"
 	},
+	"scripts": {
+		"test-js": "tests/run-tests.sh"
```

Comment:
Yeah, they're not really JS tests. Maybe I should do a followup to move these into a "misc" tests job?